packages/contracts/src/contracts/current/protocol/Exchange/mixins/MExchangeCore.sol

```diff
@@ -120,4 +120,16 @@ contract MExchangeCore is
         internal
         pure
         returns (LibFillResults.FillResults memory fillResults);
+
+    /// @dev Settles an order by transferring assets between counterparties.
+    /// @param order Order struct containing order specifications.
+    /// @param takerAddress Address selling takerAsset and buying makerAsset.
+    /// @param fillResults Amounts to be filled and fees paid by maker and taker.
+    function settleOrder(
+        LibOrder.Order memory order,
+        address takerAddress,
+        LibFillResults.FillResults memory fillResults
+    )
+        internal;
```

Comment:
The `settleOrder` can be considered 'internal' to `MixinExchangeCore` and does not have to be exposed here.